src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+public class Money {
+    private final long money;
+    public Money(long money) {
+        checkValidation(money);
+        this.money = money;
+    }
+
+    private static void checkValidation(long money) {
+        if(money < 0 ) {
```

Comment:
```suggestion
        if (money < 0) {
```

IDE에서 formatting 기능을 활용해보는 것은 어떨까요? 🤔